components/director/internal/domain/api/service.go

```diff
@@ -117,6 +146,29 @@ func (s *service) Create(ctx context.Context, applicationID string, in model.API
 	return id, nil
 }
 
+func (s *service) CreateToPackage(ctx context.Context, packageID string, in model.APIDefinitionInput) (string, error) {
```

Comment:
What do you think about a name `CreateInPackage`? And also please add TODO comment that eventually we will replace current Create method with this logic.